test/integration/component/test_network_permissions.py

```diff
@@ -758,3 +758,24 @@ def test_04_deploy_vm_for_other_user_and_test_vm_operations(self):
         command = """self.reset_network_permission({apiclient}, self.user_network, expected=True)"""
         self.exec_command("self.otheruser_apiclient", command, expected=False)
         self.exec_command("self.user_apiclient", command, expected=True)
+
+    @attr(tags=["advanced"], required_hardware="false")
+    def test_05_list_networks_under_project(self):
+        """ Testing list networks under a project """
+        self.create_network_permission(self.apiclient, self.user_network, self.domain_admin, self.project, expected=True)
+        self.list_network(self.apiclient, self.domain_admin, self.user_network, self.project, None, expected=True)
+
+        self.remove_network_permission(self.apiclient, self.user_network, self.domain_admin, self.project, expected=True)
+        self.list_network(self.apiclient, self.domain_admin, self.user_network, self.project, None, expected=False)
+
+    @attr(tags=["advanced"], required_hardware="false")
+    def test_06_list_networks_under_account(self):
+        """ Testing list networks under an account """
+        self.create_network_permission(self.apiclient, self.user_network, self.domain_admin, None, expected=True)
```

Comment:
@harikrishna-patnala 
would it be good to create network permission to `otheruser` (not `domain_admin`) as well ?

Reply:
added few more checks in the same test case @weizhouapache